packages/object-selector/README.md

```diff
@@ -0,0 +1,152 @@
+# object-selector
+
+This is a small library for selecting objects.
```

Comment:
This would be a great place to provide the context I lacked and asked for during our meeting?  Who uses this code?  When?  Why?  How?  What are the inputs and outputs?

Reply:
Thanks, I updated the README. PTAL